packages/site/src/components/PaymentChannelDetails.tsx

```diff
@@ -1,7 +1,172 @@
-type Props = {
-  paymentChannel: string;
-};
+import {
+  Box,
+  LinearProgress,
+  Typography,
+  Stack,
+  Button,
+  SvgIcon,
+} from "@mui/material";
+import { FC, useEffect, useMemo, useState } from "react";
+import { makeStyles } from "tss-react/mui";
+import { PhoneArrowUpRightIcon, UserIcon } from "@heroicons/react/24/outline";
 
-export default function PaymentChannelDetails({ paymentChannel }: Props) {
-  return <p>{paymentChannel}</p>;
+interface PaymentChannelDetails {
+  channelID: string;
+  counterparty: string; // address for now - can abstract to something richer later
+  capacity: number; // total value locked in channel
+  myBalance: number; // balance of the viewing participant
+  status: "prefund" | "running" | "unresponsive-peer" | "under-challenge";
```

Comment:
The status for a payment channel get defined in the nitro-rpc-client [here](https://github.com/statechannels/nitro-gui/blob/ecebfc8ea691f1f934e7c63bbe4b8380c36859b1/packages/nitro-rpc-client/src/types.ts#L230-L231). Could we make use of those status instead of harcoding some here?

Reply:
Sure